File: CPP/Film.cpp
#include <iostream>
#include <string>

using namespace std;

class Film {
private:
    string id_film;
    string judul;
    string genre;
    string sutradara;

public:
    // Konstruktor
    Film(string id, string jud, string gen, string sut) {
        id_film = id;
        judul = jud;
        genre = gen;
        sutradara = sut;
    }

    // Getters
    string getId() { return id_film; }
    string getJudul() { return judul; }
    string getGenre() { return genre; }
    string getSutradara() { return sutradara; }

    // Setters
    void setId(string id) { id_film = id; } // Tambahan setter ID
    void setJudul(string jud) { judul = jud; }
    void setGenre(string gen) { genre = gen; }
    void setSutradara(string sut) { sutradara = sut; }
};

File: Java/Film.java
public class Film {
    private String idFilm;
    private String judul;
    private String genre;
    private String sutradara;

    public Film(String idFilm, String judul, String genre, String sutradara) {
        this.idFilm = idFilm; 
        this.judul = judul; 
        this.genre = genre; 
        this.sutradara = sutradara;
    }

    // Getters
    public String getId() { return idFilm; }
    public String getJudul() { return judul; }
    public String getGenre() { return genre; }
    public String getSutradara() { return sutradara; }

    // Setters
    public void setId(String idFilm) { this.idFilm = idFilm; } // Tambahan setter ID
    public void setJudul(String judul) { this.judul = judul; }
    public void setGenre(String genre) { this.genre = genre; }
    public void setSutradara(String sutradara) { this.sutradara = sutradara; }
}